db/queries/assessment_records/queries.py

```diff
@@ -283,4 +283,14 @@ def update_status_to_completed(application_id):
         {AssessmentRecord.workflow_status: Status.COMPLETED}, synchronize_session=False)
 
     db.session.commit()
-    
\ No newline at end of file
+
+def update_status_to_qa_complete(application_id):
```

Comment:
Can we combine this with the function above e.g. `update_status` and pass the status enum in as an argument?

Reply:
done!